crates/codec/src/receipt/call.rs

```diff
@@ -16,19 +16,19 @@
 //!  On success (`is_success = 0`)
 //!  See [error.rs][./error.rs]
 
-use svm_types::CallReceipt;
-
 use std::io::Cursor;
 
+use svm_types::CallReceipt;
+
```

Comment:
I have been placing `svm` -related imports before `std` this whole time. What convention should we settle on?

Reply:
I've thought the order should be:

1. 3rd-party dependencies
2. `std`
3. Workspace crates
4. current crate

Reply:
Okay, I missed the workspace crates convention. I thought we packed them together with 3rd party's, I will make sense not to :+1: